crates/ruff_workspace/src/options.rs

```diff
@@ -361,6 +361,7 @@ pub struct Options {
         line-length = 120
         "#
     )]
+    #[serde(alias = "line-width", alias = "max-line-length")]
```

Comment:
Is `max-line-length` for convenience, or something we once supported?

Reply:
That was a test-only change that I accidentally committed... I reverted the change